hermes-api/src/main/java/pl/allegro/tech/hermes/api/SubscriptionHealthProblem.java

```diff
@@ -0,0 +1,116 @@
+package pl.allegro.tech.hermes.api;
+
+public enum SubscriptionHealthProblem {
```

Comment:
I think that the actual problem is that Subscription is used both as an API and as a domain object. From a domain object I would expect some useful behaviour and API I would expect to be stable. But the thing is that our API is JSON over HTTP and not Java-RPC.